packages/typegpu/tests/std/boolean/all.test.ts

```diff
@@ -0,0 +1,26 @@
+import { describe, expect, it } from 'vitest';
+import { vec2b, vec3b, vec4b } from '../../../src/data';
+import { all } from '../../../src/std/boolean';
+
+describe('all', () => {
+  it('calculates for 2 element vectors', () => {
+    expect(all(vec2b(false, false))).toBeFalsy();
```

Comment:
if `all` returned e.g. `undefined`, this assertion would pass. I'd prefer `.toBe(false)` instead of `.toBeFalsy()` and `.toBe(true)` instead of `.toBeTruthy()`. same everywhere